repository/p.json

```diff
@@ -922,6 +922,16 @@
 				}
 			]
 		},
+		{
+			"name": "Pest PHP Snippets",
+			"details": "https://github.com/geneowak/Pest-PHP-Sublime-Text-Snippets",
+			"releases": [
```

Comment:
Please add the ["snippets" label](https://packagecontrol.io/browse/labels/snippets) to make your package easier to find.